### ISSUE TITLE
'--v8-options' not returning properly in CircleCI

### ISSUE THREAD

**Issue by peterp** [gulpjs/v8flags#53]
I'm trying to use babel-node on CircleCI, but for some reason `getFlags(cb)` isn't returning the '--v8-options' properly:

```
/home/circleci/project/packages/upload-to-helpdocs/node_modules/v8flags/index.js:86
    var flags = result.match(/\s\s--[\w-]+/gm)
    ^

TypeError: Unable to get property 'map' of undefined or null reference
```

https://github.com/gulpjs/v8flags/blob/eeb439ac94301e05608f82f652bfa79f628d5b8b/index.js#L86

I'm not sure if this is an issue with Docker/ CircleCI/ or `babel-node`, so my apologies if this is an annoying issue.

**Comment by phated** [comment]
It looks like somehow the result isn't matching the expected output. I have a few more questions.

* What version of node are you using?
* Are you relying on the output of v8flags or is this part of something like gulp?
* If it's happening as part of gulp, are you trying to respawn with any node flags?  I'm thinking I might just noop the result.

**Comment by peterp** [comment]
- I'm using node `v10.13.0`
- I'm not relying on it specifically; I think `babel-node` is using it somewhere, as I saw it as a dependency:

https://github.com/babel/babel/blob/1969e6b6aa7d90be3fbb3aca98ea96849656a55a/packages/babel-node/package.json#L27

https://github.com/babel/babel/blob/1969e6b6aa7d90be3fbb3aca98ea96849656a55a/packages/babel-node/src/babel-node.js#L45-L65

**Comment by phated** [comment]
Alright.  I'm just going to return an empty array.  It's not exactly correct but it's better than crashing.

**Comment by phated** [comment]
I'm running into CI issues with my additional test.  Will try to get this fixed soon.

**Comment by peterp** [comment]
Thank you so much <3

**Comment by phated** [comment]
@peterp just released as v3.1.3 - please update your dependencies (assuming they follow semver). Also make sure to update or remove any lockfiles.
